### PR TITLE
Provide metering labels for APICurito operator and generator

### PR DESCRIPTION
**Pull request by claudio4j**
https://issues.redhat.com/browse/ENTESB-14399

### CODE REVIEW

File: apicurito/pkg/resources/deplomentconfig.go [hunk 1]
```diff
@@ -35,6 +37,16 @@ import (
 func apicuritoDeployment(c *configuration.Config, a *v1alpha1.Apicurito) (dep resource.KubernetesResource) {
 	// Define a new deployment
 	var dm int32 = 420
+	deployLabels := map[string]string{
+		"app":           "apicurito",
+		"com.company":   "Red_Hat",
+		"rht.prod_name": "1111Red_Hat_Integration " + sv.MajorMinor(version.Version),
```
Comment (lgarciaaco) [inline apicurito/pkg/resources/deplomentconfig.go:43]: Are `1111` normal?

Comment (claudio4j) [reply apicurito/pkg/resources/deplomentconfig.go]: Fixed. Leftovers from Friday testing. Thanks.

File: apicurito/deploy/bundle/container.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 annotations:
-  operators.operatorframework.io.bundle.channel.default.v1: apicurito-v0.2
-  operators.operatorframework.io.bundle.channels.v1: apicurito-v0.2
+  operators.operatorframework.io.bundle.channel.default.v1: apicurito-v7.8
+  operators.operatorframework.io.bundle.channels.v1: apicurito-v7.8
```
Comment (heiko-braun) [inline apicurito/deploy/bundle/container.yaml:3]: channel should be `apicurito-7.8.x`

Comment (heiko-braun) [reply apicurito/deploy/bundle/container.yaml:3]: See https://docs.google.com/document/d/11wV1H4a1kPdz6Mkq7DSLqkncKqOpeIiDHhyTx2QvG0s/edit

Comment (claudio4j) [reply apicurito/deploy/bundle/container.yaml]: Fixed in deploy/bundle/metadata/annotations.yaml and deploy/bundle/Dockerfile too. Thanks.

**Review by lgarciaaco — APPROVED** [review]

**Pull request merged**